setup.py

```diff
@@ -15,7 +15,7 @@
     package_data={'bluesky': ['schema/*.json']},
     install_requires=['jsonschema', 'traitlets', 'cycler',
                       'numpy', 'matplotlib', 'super_state_machine',
-                      'historydict'],
+                      'historydict', 'event_model'],
```

Comment:
The advantage of including `event-/_model` in `install_requires`, even though it's not on PyPI, is that it blows up at install time instead of at run time. This not a very big advantage, so I think we can remove it for now. We are inconsistent with `install_requires` in all our projects (e.g., databroker, to name one). I will remove `event_model`. Perhaps we should revisit the PyPI question offline.